CMakeLists.txt

```diff
@@ -468,7 +468,6 @@ if(EMSCRIPTEN)
   target_link_libraries(binaryen_wasm "-msign-ext")
```

Comment:
I think sign-ext is the default these days so you can probably remove this line too